docs/achtergronddocumentatie/artefacten/8315.md

```diff
@@ -1,10 +1,10 @@
 ---
 layout: page-with-side-nav
-title: B8315 - Naamgeving Partij, Persoon en Organisatie
-date: 31-03-2023
+title: B8315 - Naamgeving objecten Partij, Persoon en Organisatie
```

Comment:
Ik weet niet of toevoeging van het begrip 'object' helpt. Buiten het SIM noemen spreken we misschien over 'concepten', en straks zitten ze in een API en noemen we objecten 'resources'. Maar we bedoelen telkens hetzelfde...

Reply:
Ik draai dit terug. Ik dacht dat ik besluit scherper maakte, maar dat is dus niet zo.